apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/GetTableOptions.cs

```diff
@@ -22,8 +22,19 @@ namespace Google.Cloud.BigQuery.V2
     /// </summary>
     public sealed class GetTableOptions
     {
+        /// <summary>
+        /// Comma separated list of schema fields to return.
+        /// When set allows to obtain a partial table schema.
+        /// If not set, all fields, that is, the full table schema, will be returned.
```

Comment:
I'd probably simplify this to "When this property is null, the full table schema is returned."